packages/search-types/src/schemas/result.schema.ts

```diff
@@ -24,10 +24,10 @@ export const ResultSchema: Result = {
     value: expect.nullOr(Number)
   },
   tagging: {
-    query: TaggingSchema,
-    click: TaggingSchema,
-    add2cart: TaggingSchema,
-    checkout: TaggingSchema
+    query: TaggingInfoSchema,
```

Comment:
I've created a [task](https://searchbroker.atlassian.net/browse/EX-5038) to remove this, as it shouldn't be necessary anymore.